includes/admin.form.inc

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * @file
+ * Admin form for this module.
+ */
+
+/**
+ * Admin form for presbydora.
+ *
+ * @param array $form
+ *   Form object.
+ * @param array $form_state
+ *   Form values.
+ *
+ * @return array
+ *   A renderable array available as a multi-select box.
+ */
+function presbydora_admin_form(array $form, array &$form_state) {
+  $tableselect = presbydora_build_tableselect();
+  $form['presbydora_access_bypass_pids'] = array(
+    '#type' => 'tableselect',
+    '#header' => $tableselect['header'],
+    '#options' => $tableselect['rows'],
+    '#multiple' => TRUE,
+    '#default_value' => variable_get('presbydora_access_bypass_pids', ''),
```

Comment:
All Drupal variables should be removed in the .install file. An example: https://github.com/willtp87/bhofdora/blob/7.x/bhofdora.install#L24